pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/filter/query/BaseAbstractQueryBuilder.java

```diff
@@ -90,7 +95,23 @@ protected void buildUpQuery(List<String> filterValues, FilterStrategy filterStra
             baseQueryBuilder = BaseQueryBuilder.of(queryPayload);
             List<QueryBuilder> queryBuilders =
                     QueryStrategyFactory.create(new QueryStrategyFactoryPayload(baseQueryBuilder, operator, parser)).build();
-            filterStrategy.build(boolQueryBuilder, baseQueryBuilder.build(queryBuilders));
+
+            if (StringUtils.isNotBlank(path)) {
+                if (!nestedQueryBuilderLists.containsKey(path)) {
+                    nestedQueryBuilderLists.put(path, queryBuilders);
+                    correctBaseQueryBuilder.put(path, baseQueryBuilder);
+                } else {
+                    List<QueryBuilder> queryBuildersFromMap = nestedQueryBuilderLists.get(path);
+                    queryBuildersFromMap.addAll(queryBuilders);
```

Comment:
I could be confused, but shouldn't you add that again to the hashmap?
```nestedQueryBuilderLists.put(path, queryBuildersFromMap);```

Reply:
nope